packages/victory-candlestick/src/victory-candlestick.js

```diff
@@ -113,6 +113,10 @@ class VictoryCandlestick extends React.Component {
     return !!this.props.animate;
   }
 
+  shouldRenderDatum(datum) {
+    return datum._x !== null && datum._x !== undefined;
```

Comment:
I think this should check on `_high`, `_low`, `_close` and `_open` to match the behavior of the other components which also exclude null y values.